app/views/admin/promotional_feature_items/_promotional_feature_item.html.erb

```diff
@@ -1,26 +1,105 @@
-<%= content_tag_for(:section, promotional_feature_item, class: 'well') do %>
-  <% unless promotional_feature_item.image.s300.url.blank? %>
-    <%= image_tag promotional_feature_item.image.s300.url, alt: promotional_feature_item.image_alt_text %>
-  <% end %>
-
-  <% if promotional_feature_item.youtube_video_url.present? %>
-    <p>
-      <%= link_to promotional_feature_item.youtube_video_url, promotional_feature_item.youtube_video_url %>
-    </p>
-  <% end %>
-
-  <% if promotional_feature_item.title.present? %>
-    <p><%= link_to_unless promotional_feature_item.title_url.blank?, promotional_feature_item.title, promotional_feature_item.title_url %></p>
-  <% end %>
-  <p>
-    <%=format_with_html_line_breaks promotional_feature_item.summary %>
-  </p>
-  <p>
-    <% promotional_feature_item.links.each do |link| %>
-      &mdash; <%= link_to link.text, link.url %><br />
-    <% end %>
-  </p>
-
-  <%= link_to "Edit", edit_admin_organisation_promotional_feature_item_path(@organisation, @promotional_feature, promotional_feature_item), class: "btn btn-default"%>
-  <%= link_to "Delete", confirm_destroy_admin_organisation_promotional_feature_item_path(@organisation,@promotional_feature, promotional_feature_item), class: 'btn btn-danger btn-sm' %>
-<% end %>
+<%= render "components/summary_card_component", {
+  title: "Feature item #{promotional_feature_item.id}",
+  rows:
+    [
+      *(
+        if promotional_feature_item.title && !promotional_feature_item.title.blank?
+          [{
+             key: "Item title",
+             value: promotional_feature_item.title
+           }]
+        end),
+
+      *(
+        if promotional_feature_item.title_url
```

Comment:
we also need to apply that same logic here as it's currently rendering an empty row 
<img width="435" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/f0b8b208-3fb9-4b84-857b-913e74a67efe">


```
promotional_feature_item.title_url.present?
```